.github/workflows/ci.yml

```
@@ -102,6 +105,11 @@ jobs:
         if: matrix.TEST_FULL == 1
         run: |
             pip install -q -U simpleitk astropy
+      - name: Downgrade PyAV
+        # PyAV 10.0.0 is needed on py3.11 but has issues, so we only use it on
+        # py3.11 for now
+        if: matrix.PYAV_10_0_0 != 1
+        run: pip install -U av<=10.0.0
```

pip has 5 different --uxxx options so it is more self-documenting to be explicit.
```suggestion
        run: pip install --upgrade av<=10.0.0
```

---

It is also possible to:
pip install --upgrade av<10.0.0; ; python_version < '2.11'
pip install --upgrade av>=10.0.0; ; python_version >= '2.11'